2015-Fall/oPolo/Calculator/Calculator/ViewController.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  ViewController.swift
+//  Calculator
+//
+//  Created by Jonas Bruun Jacobsen on 11/07/15.
+//  Copyright (c) 2015 Jonas Bruun Jacobsen. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    
+    @IBOutlet weak var display: UILabel!
+    @IBOutlet weak var typingHistory: UILabel!
+    
+    var userIsInTheMiddleOfTypingANumber: Bool = false
```

Comment:
As it's perfectly readable from the line which type ``userIsInTheMiddleOfTypingANumber`` has, it's fine to omit ``: Bool``.